pages/places/index.js

```diff
@@ -46,7 +54,15 @@ export default function OwnPlaces({ getUniqueValues }) {
     <>
       <Wrapper700>
         <StyledPrimarySection></StyledPrimarySection>
-        <AddPlaces getUniqueValues={getUniqueValues} />
+        <AddPlaces
+          isImage={isImage}
+          setIsImage={setIsImage}
+          publicId={publicId}
+          setPublicId={setPublicId}
+          getUniqueValues={getUniqueValues}
+          isUploading={isUploading}
+          setIsUploading={setIsUploading}
```

Comment:
As mentioned before: Try to avoid  passing state setter functions as props. It's better to encapsulate them within handler functions.